text/0000-ember-data-javascript-module-api.md

```diff
@@ -0,0 +1,130 @@
+| `DS.FilteredRecordArray` | `import FilteredRecordArray from '@ember-data/store/src/filterd-record-array'` |
+| `DS.ManyArray` | `import ManyArray from '@ember-data/store/src/many-array'` |
+| `DS.RecordArray` | `import RecordArray from '@ember-data/store/src/record-array'` |
+| `DS.RecordArrayManager` | `import RecordArrayManager from '@ember-data/store/src/record-array-manager'` |
+| `DS._initializeStoreService` | `import { initializeStoreService } from '@ember-data/store'` |
+| `DS.normalizeModelName` | `import { normalizeModelName } from '@ember-data/store'` |
+| `DS._setupContainer` | `import { setupContainer } from '@ember-data/store'`
+
+### `@ember-data/model`
+| Global                                | Module                                                                      |
+|---                                    | ---  |
+| `DS.Model` | `import Model from '@ember-data/model'` |
+| `DS.Errors` | `import Errors from '@ember-data/model/src/errors'` |
+| `DS.InternalModel` | `import InternalModel from '@ember-data/model/src/internal-model'` |
+| `DS.Relationship` | `import Relationship from '@ember-data/model'` |
+| `DS.RootState` | `import RootState from '@ember-data/model/src/root-state'` |
+| `DS.Shapshot` | `import Snapshot from '@ember-data/model/src/snapshot'` |
+| `DS.attr` | `import { attr } from '@ember-data/model'`|
+| `DS.belongsTo` | `import { belongsTo } from '@ember-data/model'`|
+| `DS.hasMany` | `import { hasMany } from '@ember-data/model'` |
+
+
+### `@ember-data/adapter`
+| Global                                | Module                                                                      |
+|---                                    | ---  |
+| `DS.Adapter` | `import Adapter from '@ember-data/adapter'` |
+| `DS.AbortError` | `import AbortError from '@ember-data/adapter/src/abort-error'` |
+| `DS.AdapterError` | `import AdapterError from '@ember-data/adapter/src/adapter-error'` |
+| `DS.BuildURLMixin` | `import BuildURLMixin from '@ember-data/adapter/src/build-url-mixin'` |
+| `DS.InvalidError` | `import InvalidError from '@ember-data/adapter/src/invalid-error'` |
+| `DS.JSONAPIAdapter` | `import JSONAPIAdapter from '@ember-data/adapter/src/json-api'` |
+| `DS.RESTAdapter` | `import RESTAdapter from '@ember-data/adapter/src/rest'` |
+| `DS.TimeoutError` | `import TimeoutError from '@ember-data/adapter/src/timeout-error'` |
+| `DS.errorsHashToArray` | `import { errorsHashToArray } from '@ember-data/adapters'` |
+| `DS.errorsArrayToHash` | `import { errorsArrayToHash } from '@ember-data/adapters'` |
+
+### `@ember-data/serializer`
+| Global                                | Module                                                                      |
+|---                                    | ---  |
+| `DS.Serializer` | `import Serializer from '@ember-data/serializer'` |
+| `DS.EmbeddedRecordsMixin` | `import EmbeddedRecordsMixin from '@ember-data/serializer/src/embedded-record-mixin'` |
```

Comment:
This should be `@ember-data/serializer/src/embedded-records-mixin` (note records as plural)